bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ImageImpl.java

```diff
@@ -168,6 +168,10 @@ protected void initModel() {
                 hasContent = false;
                 return;
             }
+            // The jcr:mimeType property may contain a charset prefix (image/jpeg;charset=UTF-8).
```

Comment:
I'd call it `charset suffix` or `charset property`.

Reply:
thanks @jckautzmann - spotted my error too, updated